Dockerfile

```diff
@@ -1,19 +1,34 @@
-FROM alpine:3.3
+FROM alpine:edge
```

Comment:
We've had trouble with `edge` before as it changes often. Would the `3.4` tag work?